Alicerce/Sources/Logging/Log+ItemJSONFormatter.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Log+ItemJSONFormatter.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class ItemJSONFormatter: LogItemFormatter {
```

Comment:
Agree, I also think we don't need to repeat `public` since the parent scope is already public